application/event_processor/dos.py

```diff
@@ -1,24 +1,21 @@
+from os import environ
 from logging import getLogger
-from os import environ, getenv
-from typing import List
-
-from boto3 import client
-from change_request import (
-    ADDRESS_CHANGE_KEY,
```

Comment:
Aren't these constants required?

Reply:
Not here, see event_processor file

Reply:
To add, the code written to compare changes has been moved from the dos module to the event processor file. As its not a dos specific set of functions I felt it made more logical sense to move it. Not sure why I originally wrote it in dos.py

And naturally the constants they use are no longer needed in this file.